handbook/network-security/network-security.md

```diff
@@ -0,0 +1,225 @@
+# Network Security
+
+SystemLink connects test systems to a central server to aggregate data for monitoring and analysis. SystemLink uses industry standard tools and best practices to ensure the SystemLink server and test systems utilize secure networking techniques.
+
+Refer to [Workspaces and Role-based Access Control](/rbac/rbac) for details secure user interactions with SystemLink resources. Refer to [Single Sign-on with OpenID Connect](/openid-connect/openid-connect/) and [Sign on with LDAP](/ldap/ldap/) for details on securely authenticating with SystemLink.
+
+## Network diagram
+
+<figure>
+  <img src="../../img/network-diagram.png" width="500" />
```

Comment:
Since it's enabled by default, might be worth mentioning AMQP here and linking to the deprecation section.